### ISSUE TITLE
Docker does not listen

### ISSUE THREAD

**Issue by LoLei** [insomniacslk/irc-slack#106]
### Problem
```
$ docker run insomniacslk/irc-slack
time="2020-04-29T16:02:00Z" level=info msg="Setting log level to 'info'" prefix=main
time="2020-04-29T16:02:00Z" level=info msg="Starting server on 127.0.0.1:6666" prefix=main
time="2020-04-29T16:02:00Z" level=info msg="Listening on 127.0.0.1:6666" prefix=main
```

However, the Docker container process does not listen on port 6666, as stated in the log.

```
$ sudo ss -tulwn | grep 6666
<no results>
```

If I use the git repo and build and run it myself it works:
```
$ ./irc-slack
[2020-04-29T17:55:32+02:00]  INFO main: Setting log level to 'info'
[2020-04-29T17:55:32+02:00]  INFO main: Starting server on 127.0.0.1:6666
[2020-04-29T17:55:32+02:00]  INFO main: Listening on 127.0.0.1:6666
```
```
$ sudo ss -tulwn | grep 6666
tcp   LISTEN 0      4096       127.0.0.1:6666       0.0.0.0:*
```

**Comment by insomniacslk** [comment]
The README is missing the `-p` argument on the docker command line, I'll add it in a bit. Basically just add `-p 6666:6666`, the first is the port bound in the container, the latter is what you will see on the host side, and you can use whatever you want there

**Comment by insomniacslk** [comment]
@LoLei can you check the fix at #107 ?

**Comment by LoLei** [comment]
I'm building the new Dockerfile right now.
